### ISSUE TITLE
transition: Accessibility map. Add calculation properties to exported files

### ISSUE THREAD

**Issue by tahini** [chairemobilite/transition#189]
When exporting data from the accessibility map, the parameters used for the calculation should be available in the exported results.
